modules/data-streams/src/test/java/org/elasticsearch/datastreams/DataStreamIndexSettingsProviderTests.java

```diff
@@ -105,7 +109,7 @@ public void testGetAdditionalIndexSettingsIndexRoutingPathAlreadyDefined() throw
                 "_doc": {
                     "properties": {
                         "field1": {
-                            "type": "keyword"
+                            "type": "keyword",
```

Comment:
I can't believe this didn't cause previous build failures.

Reply:
Me, too but I didn't investigate further.